code_challenges.rb

```diff
@@ -31,3 +54,18 @@
 # Expected output: 'LEARN is not a palindrome'
 is_palindrome3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+
+def palindrome(string)
+  if string.downcase == string.reverse
+    "#{string} is a palindrome"
+  else
+    "#{string} is not a palindrome"
+  end
+end
+
+p palindrome(is_palindrome1)
+p palindrome(is_palindrome2)
+p palindrome(is_palindrome3)
+
+# // the above method does not work. All 3 outputs state "...is not a palindrome". It doesn't seem to be getting past the capitalization in Racecar and Rotator since those should return with the output "...is a string"
```

Comment:
great observation! And you are totally correct its getting hung up on the capitalization! If you follow that down the rabbit hole you can make a small change to your evaluation logic and ...
```ruby
def palindrome(string)
  if string.downcase == string.downcase.reverse
    "#{string} is a palindrome"
  else
    "#{string} is not a palindrome"
  end
end
``` 

Boom you got it!